Objects/memoryobject.c

```diff
@@ -1164,8 +1164,8 @@ static int
 memory_clear(PyObject *_self)
 {
     PyMemoryViewObject *self = (PyMemoryViewObject *)_self;
-    (void)_memory_release(self);
-    Py_CLEAR(self->mbuf);
+    if(_memory_release(self) == 0)
+        Py_CLEAR(self->mbuf);
```

Comment:
We should not silently ignore _memory_release() exception. It should be either cleared or logged. I suggest to log the exception:

```suggestion
    if(_memory_release(self) == 0) {
        Py_CLEAR(self->mbuf);
    }
    else {
        PyErr_FormatUnraisable("Exception ignored in memoryview clear");
    }
```

Reply:
Thx for review.